HelpCentre/QA/HelpCentre: Intro to HelpCentre.md

```diff
@@ -0,0 +1,1221 @@
+        "inbound_id": "@inbound_id",
+        "feedback": {
+          "feedback_type": "negative",
+          "faq_id": "@body['@selected_faq'].id"
+        }
+      }
+
+      """,
+      headers: [
+        ["Authorization", "Token @global.settings.mc_ndoh_hub_token"],
+        ["content-type", "application/json"]
+      ]
+    )
+
+  log("Feedback result = @feedback_result")
+end
+
+card AcknowledgePositive do
+  log("AcknowledgePositive")
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_acknowledgement_positive_"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  ack_pos_msg = page.body.body.text.value.message
+  button_labels = map(page.body.body.text.value.buttons, & &1.value.title)
+
+  buttons(
+    RouteUrgency: "@button_labels[0]",
+    HelpCentre: "@button_labels[1]",
+    MainMenu: "@button_labels[2]"
+  ) do
+    text("@ack_pos_msg")
+  end
+end
+
+card AcknowledgePositiveError, then: AcknowledgePositiveError do
+  buttons(
+    RouteUrgency: "@button_labels[0]",
+    HelpCentre: "@button_labels[1]",
+    MainMenu: "@button_labels[2]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+card AcknowledgeNegative, then: RouteUrgency do
+  log("AcknowledgeNegative")
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_acknowledgement_negative_"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  ack_neg_msg = page.body.body.text.value.message
+  text("@ack_neg_msg")
+end
+
+```
+
+# Send to Helpdesk
+
+```stack
+card SendToHelpdesk, then: CheckHelpdeskStatus do
+  # Attach the relevant labels to the chat
+  log("Query urgency is @inbound_query_urgency")
+  log("Query type label to attach is @inbound_query_type")
+  log("External ID of msg to label  = @inbound_message_id")
+end
+
+card CheckHelpdeskStatus when helpdesk_open == true, then: HelpdeskOpen do
+  log("Helpdesk status is open")
+end
+
+card CheckHelpdeskStatus, then: HelpdeskClosed do
+  log("Helpdesk status is closed")
+end
+
+card HelpdeskOpen do
+  attach_label_result =
+    post(
+      "https://whatsapp-praekelt-cloud.turn.io/v1/messages/@inbound_message_id/labels",
```

Comment:
Can we not use the `add_label` function here instead of making a call to the API? https://whatsapp.turn.io/docs/build/functions#assigning-labels-to-messages

Reply:
Unfortunately not in this case.  `add_label` adds the label to the last message received.  For this part of the flow, they want to add the label to a few message back, the original search query the user entered, so its easier for the operator to pick up from there

Reply:
We also don't want to add the label at the time that they enter the query, because then an agent might pick up the call before we've tried the rest of the flow

Reply:
Ah I understand, it's not the last received message we're adding the label to